sdk/program/src/sysvar/epoch_rewards.rs

```diff
@@ -0,0 +1,139 @@
+//! Epoch rewards for current epoch
+//!
+//! The _epoch rewards_ sysvar provide access to the [`EpochRewards`] type,
+//! which tracks the progress of epoch rewards distribution. It includes the
+//!   - total rewards for the current epoch, in lamports
+//!   - distributed rewards for the current epoch, in lamports, so far
+//!   - distribution completed block height, i.e. after this block height for distribution of all
+//!     staking rewards for the current epoch will be completed should have been completed.
+//!
+//! [`EpochRewards`] implements [`Sysvar::get`] and can be loaded efficiently without
+//! passing the sysvar account ID to the program.
+//!
+//! See also the Solana [documentation on the epoch rewards sysvar][sdoc].
+//!
+//! [sdoc]: https://docs.solana.com/developing/runtime-facilities/sysvars#epochrewards
+//!
+//! # Examples
+//!
+//! Accessing via on-chain program directly:
+//!
+//! ```no_run
+//! # use solana_program::{
+//! #    account_info::{AccountInfo, next_account_info},
+//! #    entrypoint::ProgramResult,
+//! #    msg,
+//! #    pubkey::Pubkey,
+//! #    sysvar::epoch_rewards::{self, EpochRewards},
+//! #    sysvar::Sysvar,
+//! # };
+//! # use solana_program::program_error::ProgramError;
```

Comment:
I expect this is copied from other sysvars, but can you remind me if there is a reason why this can't be part of the use statement above?

Reply:
fixed.